modules/serverless/pages/serving/serving-kourier-istio-ingresses.adoc

```diff
@@ -0,0 +1,61 @@
+= Kourier and Istio ingresses
+:compat-mode!:
+// Metadata:
+:description: Kourier and Istio ingresses
+
+{serverlessproductname} supports the following two ingress solutions:
+
+* Kourier
+* Istio using {smproductname}
+
+The default is Kourier.
+
+[[serverless-ingresses-kourier-overview]]
+== Kourier
+
+Kourier is the default ingress solution for {serverlessproductname}. It has the following properties:
+
+* It is based on envoy proxy.
+* It is simple and lightweight.
+* It provides the basic routing functionality that {serverlessproductname} needs to provide its set of features.
+* It supports basic observability and metrics.
+* It supports basic TLS termination of Knative Service routing.
+* It provides only limited configuration and extension options.
```

Comment:
I think we can delete this last line. It provides only limited configuration and extension options. as it is our default option